### ISSUE TITLE
Change Complaint Type from Case Details Page

### ISSUE THREAD

**Issue by andrew-rist-tw** [PublicDataWorks/complaint-manager#238]
blocked by #239 

### Business Context:
this is similar to #239 
if the `chooseComplaintType` flag is true then the user can choose what type the complaint is when creating the case.  We want to also be able to do that after the case is created from the Case Details Page.  So we want to add a button (MaterialUI IconButton using the "Settings" icon, which should be a little gear) next to the complaint type on the case details page sidebar (CaseDrawer) to open a dialog that lets you choose a new complaint type via dropdown (possible chance to encapsulate the functionality from the Create Case Dialog in a component that can be reused on this new dialog)

**As a** Hawaii employee
**I want** to be able to change the complaint type on the Case Details Page
**So that** if the type changes or I just made a mistake while creating it, I can change the type

### In Scope
1. Change complaint type (by calling editCase PUT handler)
2.
3.

### Out of Scope
1.

### Open Questions
1.

## Acceptance Criteria

1. Scenario: Change the complaint type
Given i am on the case details page
When I click on a button next to complaint type
Then i see a dropdown of complaint type options

2. select new complaint type
Given I see the dropdown of complaint type options
When I select one
Then the complaint type for that complaint is updated!

### Checklist (should be reviewed before grooming as the BA or Pair are writing the card—NOT reviewed for the first time at grooming)
0. Discuss the technical implementation of this card.
1. Discuss how this feature would be included in or affect the Audit Log.
2. In respect to security, what could go wrong? Brainstorm threats using STRIDE model, especially if this card introduces a new technical architecture or design pattern. (TIMEBOX: 7 mins) https://docs.google.com/document/d/18AFl1d3SRieI4d_8R3GyEazqQOycLNDHGN_vOYIHuiE/edit#heading=h.aio9zb5j63wl
3. Ensure any open questions have been resolved with stakeholders.
4. Determine if documentation should be referenced or created.
5. Ensure to include all necessary mockups to this card.
6. Discuss what success/failure user feedback messages would be present, if any occur (front end/back end).
7. Discuss why this feature would not be implemented behind a toggle / what would happen if this card was partially done while a release branch was being cut.
8. If this card is part of a sequence of cards, discuss if this card should be blocked until the testing of previous  cards in the sequence have been finished.
9. Have we added edge cases as ACs that we're considered from a QA perspective
10. Does this card overlap with any entities outlined in [API Housekeeping](https://www.pivotaltracker.com/story/show/173963545)?  

**Notes:**
